### PR TITLE
chore: add explanations of destinations

### PR DESCRIPTION
**Pull request by Spazzy757**
# Changelog

- Sets up the destination configuration documentation

### CODE REVIEW

File: content/study-configuration/destination.md [hunk 1]
```diff
@@ -4,28 +4,44 @@ date: 2023-04-05T08:41:03+02:00
...
-Curious Learning Literacy Apps. Requires the following parameters:
+We require the following to be set:
 
-- `from`: int Starting timestamp for retrieving app events from Literacy Data API.
-- `app_id`: str The app ID for the Google Play store.
-- `facebook_app_id`: str he app ID for Facebook.
+- `Initial Shortcode`: The shortcode of the initial form users should be sent to.
+- `Survey Name`: The survey that contains all the forms with data of interest.
+
+## Typeform
```
Comment (Spazzy757) [inline content/study-configuration/destination.md]: @nandanrao is this still valid or do we now consider it a Web APP Destination with a field of `URL Template`

File: content/study-configuration/destination.md [hunk 2]
```diff
@@ -4,28 +4,44 @@ date: 2023-04-05T08:41:03+02:00
...
 - `deeplink_template`: str
-- `app_install_state`: str
-- `user_device`: list[str]
-- `user_os`: list[str]
-- `attribution_id`: str
+- `App Install State`: 
+- `User Device`:
+- `User OS`: 
+- `Attribution Id`:
+
```
Comment (Spazzy757) [inline content/study-configuration/destination.md:51]: @nandanrao can I get some more info on these fields, do we still call this the Curious Learning Literacy App or do we just call it an App destination?

File: content/study-configuration/destination.md (hunk 1 above)
Comment (nandanrao) [reply content/study-configuration/destination.md]: You are correct, it's now a Web destination with a URL template, yes!

File: content/study-configuration/destination.md (hunk 2 above)
Comment (Spazzy757) [reply content/study-configuration/destination.md:51]: @nandanrao just need more info on these and Im ready to go

Comment (nandanrao) [reply content/study-configuration/destination.md:51]: Yeah sorry, as a destination this should be generic for all apps. The only part that is unique to curious learning is as a data source.

File: content/study-configuration/destination.md [hunk 3]
```diff
@@ -4,28 +4,48 @@ date: 2023-04-05T08:41:03+02:00
 draft: false
 ---
 
-Every study needs a destination, where do the recruitment ads send the users? Destinations need to be connected to Virtual Lab so that, not only does it know where to send people who click on the recruitment ads, but also so that it knows how to collect the information about those who become study participants and knows how to optimize ads.
+Every study needs a destination, where do the recruitment ads send the users to? 
 
-Towards that end, Virtual Lab supports a set a "destinations" and is written in such a way that it is easy to add a new destination. When configuring your study, you can create one or more destinations and each destination needs a unique name (key) that you can refer to it by.
+Destinations need to be connected to Virtual Lab so that, not only does it know where to send people who click on the recruitment ads, but also so that it knows how to collect the information about those who become study participants and knows how to optimize ads.
```
Comment (nandanrao) [inline content/study-configuration/destination.md:9]: Techically, we think of the `data source`, where one collects information, as separate from the destination. 

We're having more and more instances of why that actually makes sense. 

For example, right now we have a couple new projects where they want us to screen people in the chatbot then send them to an external web survey. One can imagine that if we integrated their websurvey platform, we would have a destination being Fly Messenger, but we would have the data source being the external web survey platform.

**Review by nandanrao — CHANGES_REQUESTED** [review]

**Comment by nandanrao** [comment]
Except for that small comment, this LGTM!

**Pull request closed without merging**